sql/V009__create_signoff_table.sql

```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS signoff;
+
+CREATE TABLE signoff (
+  id SERIAL PRIMARY KEY,
+  created TIMESTAMP WITH TIME ZONE DEFAULT (NOW())::TIMESTAMP(0) WITH TIME ZONE NOT NULL,
+  case_identifier varchar NOT NULL,
+  username varchar NOT NULL,
+  qc_passed boolean NOT NULL,
+  signoff_step_name varchar NOT NULL,
+  deliverable_type varchar NOT NULL,
+  comment text
+);
+
```

Comment:
The requirements state `If a sign-off is added with the same caseIdentifier, signoffStepName, username, and deliverableType the old sign-off record will be replaced (regardless of QC status).` I think it's worth enforcing that at the table level with a unique index:
```
CREATE UNIQUE INDEX uniq_caseid_signoffstepname_deliverabletype_username ON signoff (case_identifier, signoff_step_name, deliverable_type, username);
```

Reply:
@grapigeau this appears to be +1'd and resolved but not actually implemented? did discussion on this index happen elsewhere?

Reply:
Ah I should've marked this as resolved since I did add a (similar) constraint
`CREATE UNIQUE INDEX uniq_caseid_signoffstepname_deliverabletype ON signoff (case_identifier, signoff_step_name, deliverable_type);`
It doesn't include username based on discussion in slack

Reply:
Within a case there should only be one sign-off for each step + deliverable, the person who creates that sign-off is redundant